packages/ui/src/components/va-date-picker/components/VaMonthPicker/VaMonthPicker.vue

```diff
@@ -4,8 +4,8 @@
       class="va-month-picker__month-wrapper"
       v-for="(month, monthIndex) in months"
       :key="monthIndex"
-      @mouseenter="hoveredIndex = monthIndex"
-      @mouseleave="hoveredIndex = -1"
+      @mouseenter="onMouseHover(month); hoveredIndex = monthIndex"
```

Comment:
You can actually `watch` for `hoveredIndex` ref or `hoveredValue` inside `use-picker` hook instead of adding extra function everywhere.